app/build.gradle

```diff
@@ -15,9 +15,18 @@ def versionMajor = 0
 def versionMinor = 1
 def versionPatch = 0
 
+def getGitTimetamp() {
+    def stdout = new ByteArrayOutputStream()
+    exec {
+        commandLine 'git', 'show', '-s', '--format=%ct'
+        standardOutput = stdout
+    }
+    return stdout.toString() as Integer
+}
+
 def generateVersionCodeFromTimestamp() {
     // It's unix timestamp divided by 10: It's incremented by one every 10 seconds.
```

Comment:
This comment should be updated too

Reply:
I deliberately left the comment in place, I think it's still valid. How would you change it?

Reply:
The comment can stay like that. The point is that the value doesn't change anymore if there is no new commit.